virtual.py

```diff
@@ -16,3 +16,17 @@
 def speak(text):
     engine.say(text)
     engine.runAndWait()
+    
+def wishMe():
+    hour=datetime.datetime.now().hour
+    if hour>=0 and hour<12:
+        speak("Hello,Good Morning")
+        print("Hello,Good Morning")
+    elif hour>=12 and hour<18:
+        speak("Hello,Good Afternoon")
+        print("Hello,Good Afternoon")
+    else:
+        speak("Hello,Good Evening")
+        print("Hello,Good Evening")
+    
+
```

Comment:
add codes for voice recognition also @shivamkumar0611 @sauravmandal255 @santosh-sudo 
All other members please respond on this ..